distribution/linux/flatpak/io.github.openenroth.openenroth.yml

```diff
@@ -0,0 +1,85 @@
+app-id: io.github.openenroth.openenroth
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '24.08'
+sdk: org.freedesktop.Sdk
+command: io.github.openenroth.openenroth.sh
+
+finish-args:
+  # hardware 3D and controller access
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Audio
+  - --socket=pulseaudio
+
+cleanup-commands:
+  - mkdir -p ${FLATPAK_DEST}/lib/ffmpeg
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm18
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
```

Comment:
Is ffmpeg version effectively fixed here? I'm asking b/c I don't want to find us in a sutuation when it's 2025 and this now pulls in ffmpeg 8 which breaks our build.